django/db/models/query.py

```diff
@@ -724,6 +724,8 @@ def delete(self):
         assert not self.query.is_sliced, \
             "Cannot use 'limit' or 'offset' with delete."
 
+        if self.query.distinct or self.query.distinct_fields:
+            raise TypeError("Cannot call delete() with .distinct()")
```

Comment:
```suggestion
            raise TypeError('Cannot call delete() after .distinct()')
```

Reply:
Seems like most of the code uses double quotes. Should I change this to single quotes anyway?